handbook/sales/README.md

```diff
@@ -135,6 +135,41 @@ To schedule an [ad hoc meeting](https://www.vocabulary.com/dictionary/ad%20hoc)
  - Do not assign the issue. The Director of Solutions Consulting will assign the issue.
  - Ensure that the product category is defined ("Endpoint ops", "Device management", or "Vulnerability management") in the description of the issue.
 
+### Conduct a technical evaluation (tech eval)
+
```

Comment:
Fix strange line breaks. It makes it hard to review the whole sentence at once and impossible to make suggestions.